src/Item.php

```diff
@@ -123,7 +123,7 @@ public function get()
 	 * Note: This method MUST NOT have a race condition between calling isHit()
 	 * and calling get().
 	 *
-	 * @return boolean
+	 * @return bool
```

Comment:
we have `@return boolean`, `@param boolean`, `@var boolean` in another places, do they need to be updated for consistency?